### PR TITLE
Add starting point for terminology page

### PR DESCRIPTION
**Pull request by bertramakers**
### Added

- Added starting point for terminology page. Can always be tweaked/expanded later when needed.

---

Preview: https://docs.publiq.be/docs/uitdatabank/branches/uitdatabank%2Fterminology/76b7bd22f7f51-terminology#nil-location (login required)

### CODE REVIEW

**Comment by bertramakers** [comment]
I still need to expand the organizers explanation a bit to include that they can also be related to places, and to explain dummy organizers (like dummy locations).

File: projects/uitdatabank/docs/terminology.md [hunk 1]
```diff
@@ -2,8 +2,82 @@
...
+
+An API resource with a unique id that represents a place that can be visited and/or where [events](#event) can be located.
+
+If the place is published, it will also be published in [online calendars](#online-calendar) for interested visitors to find them, or to find other events located at the same place.
+
+Places created as a draft that do not get published can still be used as [locations](#location) for events, but will not be visible in search results in online calendars.
+
+## Offer
+
+The collective name both for events and places, usually used to indicate. Translated from the Dutch "Aanbod" to indicate the events/places that users of [online calendars](#online-calendar) can attend or visit.
```
Comment (LucWollants) [inline projects/uitdatabank/docs/terminology.md:29]: The `, usually used to indicate.` part feels a bit weird.

File: projects/uitdatabank/docs/terminology.md [hunk 2]
```diff
@@ -2,8 +2,82 @@
...
+A [place](#place) that a specific [event](#event) is happening at. The place will be included in the event's JSON as a `location` property.
+
+### Nil location
+
+A location for events that have no [physical location](#physical-location), like [online events](#online-event), to maintain compatibility with older integrations that expect every [event](#event) to have a location.
+
+*   Test: https://io-test.uitdatabank.be/place/00000000-0000-0000-0000-000000000000 
+*   Production: https://io.uitdatabank.be/place/00000000-0000-0000-0000-000000000000
+
+Online events will automatically become related to the nil location. Other events cannot be located at the nil location.
```
Comment (LucWollants) [inline projects/uitdatabank/docs/terminology.md:42]: The `Other events` can maybe be defined as "offline" or "mixed"?

File: projects/uitdatabank/docs/terminology.md [hunk 3]
```diff
@@ -2,8 +2,82 @@
...
+
+An example of an event with a dummy location: https://io.uitdatabank.be/event/e160f0f3-89a8-45d7-94a6-34fda89fd69c
+
+These events were imported from UiTdatabank v2, where an event's location did not have to be a separate place that would be related.
+
+No new events can be created with a dummy location in UiTdatabank v3.
+
+## Organizer
+
+An API resource with a unique id that represents an organization or person that organizes [events](#event).
```
Comment (LucWollants) [inline projects/uitdatabank/docs/terminology.md:67]: I think a place can also have an organizer?

File: projects/uitdatabank/docs/terminology.md (hunk 1 above)
Comment (bertramakers) [reply projects/uitdatabank/docs/terminology.md:29]: Woops I forgot a part there, will fix it

**Review by LucWollants — APPROVED** [review]
I only have a few smaller remarks, so I already approved.

File: projects/uitdatabank/docs/terminology.md (hunk 2 above)
Comment (bertramakers) [reply projects/uitdatabank/docs/terminology.md:42]: Good idea!

**Review by LucWollants — APPROVED** [review]

**Pull request merged**